dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/AdaptiveCards.cs

```diff
@@ -191,13 +178,7 @@ public AdaptiveCards(Application<TState, TTurnStateManager> app)
                     throw new TeamsAIException($"Unexpected AdaptiveCards.OnActionExecute() triggered for activity type: {turnContext.Activity.Type}");
                 }
 
-                AdaptiveCard adaptiveCard = await handler(turnContext, turnState, invokeValue.Action.Data, cancellationToken);
-                AdaptiveCardInvokeResponse adaptiveCardInvokeResponse = new()
```

Comment:
Would it be better if providing something like `AdaptiveCardInvokeResponseBuilder.FromCard`?